pkg/inject/config.go

```diff
@@ -205,6 +209,8 @@ func (cfg *Config) BindFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&cfg.DualStackEndpoint, flagDualStackEndpoint, false, "Use DualStack Endpoint")
 	fs.BoolVar(&cfg.DualStackEndpoint, flagEnvoyAdminAccessEnableIpv6, false, "Enable Admin access when using IPv6")
 	fs.StringVar(&cfg.ClusterName, flagClusterName, "", "ClusterName in context")
+	fs.IntVar(&cfg.UseFipsEndpoint, flagUseFipsEndpoint, 0, "To use FIPS endpoint")
+	fs.IntVar(&cfg.UseDualStack, flagUseDualStack, 0, "To use Dual Stack Endpoint")
```

Comment:
There's already a DualStackEndpoint with an identical description above (then another flag bound to the same flag for enable IPv6 admin endpoint). I see that it's used in the Envoy sidecar configuration. How do these decisions relate to each other? Does it make sense to make a separate flag (might be necessary for backward compat)? If these should be separate I think they need their descriptions updated a bit to make them easier to understand and maybe update your new flag name to differentiate it.